addons/addon-base-rest-api/packages/base-controllers/lib/__tests__/__fixtures__/authentication-provider-configs.js

```diff
@@ -323,6 +323,7 @@ const configurations = [
 const publicConfigurations = [
```

Comment:
I don't see any oidc auth configurations in test fixtures. We need add some for testing oidc auth provider.